radicle-httpd/src/api/json.rs

```diff
@@ -129,6 +129,7 @@ struct Author {
 #[derive(Serialize)]
 #[serde(rename_all = "camelCase")]
 struct Comment {
+    id: OpId,
```

Comment:
I added `OpId` here to each comment, to have a reference in the web client to refer to, when needing to reply to a specific comment or to react.